### PR TITLE
add conditionals to support apm-server docs update

### PR DESCRIPTION
**Pull request by graphaelli**
This is to support https://github.com/elastic/apm-server/pull/1103.  While that's still in review, it's moving along and I'd like to know if there are any objections here that need to be factored in there.

### CODE REVIEW

File: libbeat/docs/command-reference.asciidoc [hunk 1]
```diff
@@ -20,11 +20,11 @@
 :modules-command-short-desc: Manages configured modules
 :run-command-short-desc: Runs {beatname_uc}. This command is used by default if you start {beatname_uc} without specifying a command
 
-ifeval::["{has_ml_jobs}"=="yes"]
+ifdef::has_ml_jobs[]
```
Comment (ruflin) [inline libbeat/docs/command-reference.asciidoc]: Can you share some background on how this is different from the previous true / false?

**Comment by ruflin** [comment]
@graphaelli In general I don't see any reason we shouldn't have that.
@dedemorton Would be good if you could have a look at this one.

File: libbeat/docs/command-reference.asciidoc (hunk 1 above)
Comment (graphaelli) [reply libbeat/docs/command-reference.asciidoc]: In `libbeat/docs/security/basic-auth.asciidoc` `has_ml_jobs` is used as `ifdef`.  It seems we typically use `ifdef` over `ifeval` with a value so I sync'd in that direction.

**Comment by graphaelli** [comment]
elastic/apm-server#1103 is now ready for merge.  Waiting on approval of these changes for a bit before merging.

**Comment by ruflin** [comment]
@graphaelli Merging as I definitively don't want to block you on this one and change looks fine.

@dedemorton Please still have a look and let me know if we need to change / revert some things.

**Comment by graphaelli** [comment]
thank you!

**Comment by dedemorton** [comment]
Looks good! The only problem I noticed was one that I introduced. See https://github.com/elastic/beats/pull/7602.

@graphaelli You'll probably want to pick up the changes in https://github.com/elastic/beats/pull/7602 after it's merged.

**Comment by graphaelli** [comment]
@dedemorton thanks for the heads up

**Pull request merged**